doc/findinfiles.rst

```diff
@@ -1,27 +1,56 @@
 :orphan:
 
-#############
-Find in Files
-#############
+####
+Find
+####
 
-The **Find in Files** pane allows to you perform full-text, recursive content search in a user-defined scope, with advanced features to filter your results.
+The **Find** pane allows you to search the content of files in a user-defined location, with advanced features to filter your results.
 
 .. image:: images/find_in_files/find_in_files_inprogress.png
    :alt: Spyder Find in Files panel, with search results shown per-file
 
 |
 
 
+
+===================
+Using the Find pane
+===================
+
+To search for text in the Find pane, write it in the top bar of the pane and press the search button.
+This will allow you to view and navigate through all the occurrences of your search text in your working directory. You can expand or contract the search results to view the results on each file.
```

Comment:
```suggestion
This will allow you to view and navigate through all the occurrences of your search text in your working directory.
You can expand or contract the search results to view the results on each file.
```